TLM/TLM/Custom/PathFinding/CustomPathManager.cs

```diff
@@ -96,8 +120,42 @@ public class CustomPathManager : PathManager {
                     Destroy(stockPathFinds[i]);
                 }
             }
+        }
+
+        public void UpdateOldPathManagerValues(PathManager stockPathManager) {
+            stockPathManager.m_simulationProfiler = m_simulationProfiler;
+            stockPathManager.m_drawCallData = m_drawCallData;
+            stockPathManager.m_properties = m_properties;
+            stockPathManager.m_pathUnitCount = m_pathUnitCount;
+            stockPathManager.m_renderPathGizmo = m_renderPathGizmo;
+            stockPathManager.m_pathUnits = m_pathUnits;
+            stockPathManager.m_bufferLock = m_bufferLock;
```

Comment:
Hmm, are you sure that all fields are necessary?
`m_bufferLock`, `m_pathUnits`, `m_properties` and `m_simulationProfiler` are references, we already copied them from stockPathManager at creation (`UpdateWithPathManagerValues()`).